io/driver/module_unload_load.py.data/module_unload_load.yaml

```diff
@@ -0,0 +1,3 @@
+Parameters:
+    module: ""
+    iteration: 10 
```

Comment:
The existing test was having a functionality wherein only pci modules were being unloaded/loaded.
Can you add that functionality also here please ?